internal/internal_event_handlers.go

```diff
@@ -1269,7 +1316,7 @@ func (weh *workflowExecutionEventHandlerImpl) handleRequestCancelExternalWorkflo
 	// for cancellation of external workflow, we have to use cancellation ID
 	attribute := event.GetRequestCancelExternalWorkflowExecutionInitiatedEventAttributes()
 	workflowID := attribute.WorkflowExecution.GetWorkflowId()
-	cancellationID := string(attribute.Control)
+	cancellationID := attribute.Control
```

Comment:
I think control is no longer needed.  At least we should create a separate issue to see if we can get rid of Control as the identifier.

Reply:
We had this discussion before and decided to leave it as `string`. Here, I just removed redundant type conversion.